auth/src/main/res/layout/idp_button_google.xml

```diff
@@ -1,6 +1,6 @@
 <Button
     xmlns:android="http://schemas.android.com/apk/res/android"
     style="@style/FirebaseUI.Button.AccountChooser.GoogleButton"
-    android:text="@string/sign_in_with_google"
+    android:text="@string/common_signin_button_text_long"
```

Comment:
Where is this defined, in Google Play Services?  I don't see it in `strings.xml`.  If it's defined in a dependency let's re-define it here so we don't get changing text over time, the string resources of Google Play Services are not considered a public API and may change or go away.